lib/github.js

```diff
@@ -105,15 +115,19 @@ Organization.prototype.parseLinks = function(response) {
 }
 
 Organization.prototype.clone = function(repo) {
+
     if (this.exclude && this.regexp && repo.name.match(this.exclude)) {
         return cli.debug('Repository ' + repo.name + ' ignored, exclude option: ' + this.exclude);
     } else if (this.exclude && !this.regexp && this.exclude.indexOf(repo.name) != -1) {
         return cli.debug('Repository ' + repo.name + ' ignored, exclude option: ' + this.exclude);
     } else if (this.only && this.regexp && !repo.name.match(this.only)) {
-        cli.debug('Repository ' + repo.name + ' ignored, only option: ' + this.only);
+        return cli.debug('Repository ' + repo.name + ' ignored, only option: ' + this.only);
     } else if (this.only && !this.regexp && this.only.indexOf(repo.name) == -1) {
         return cli.debug('Repository ' + repo.name + ' ignored, only option: ' + this.only);
-    } else {
+    } else if (this.existingDirs && this.existingDirs.indexOf(repo.name) > -1) {
+        return cli.info('Repository ' + repo.name + ' ignored, directory already found');
+    }
+    else {
```

Comment:
Can you leave this `else` in the previous line? Just a coding style preference 😄